src/containers/Tabs/Account.tsx

```diff
@@ -153,7 +153,7 @@ const AccountTabContainer: React.FC<AccountTabContainerProps> = ({ isOpen }) =>
         deleteTransaction={deleteTransaction}
         isValid={isValidAccount}
         isOpen={isOpen}
-        isReadOnly={isReadOnly}
+        isReadOnly={isReadOnly!}
```

Comment:
what does the ! mean here?

Reply:
To ignore that isReadOnly could be null or undefined